charmcraft/commands/extensions.py

```diff
@@ -47,17 +47,18 @@ def run(self, parsed_args: argparse.Namespace):
         """Print the list of available extensions and their bases."""
         extension_presentation: dict[str, ExtensionModel] = {}
 
-        for extension_name in extensions.registry.get_extension_names():
-            extension_class = extensions.registry.get_extension_class(extension_name)
+        for extension_name in extensions.get_extension_names():
```

Comment:
They are the same thing?

Reply:
Yep, but because `extensions` doesn't export `registry`, pyright complains.